src/main/resources/org/jenkinsci/plugins/github_branch_source/Messages.properties

```
@@ -66,3 +66,5 @@ ExcludeArchivedRepositoriesTrait.displayName=Exclude archived repositories
 
 GitHubSCMNavigator.general=General
 GitHubSCMNavigator.withinRepository=Within repository
+
+GitHubAppCredentials.displayName=GitHub app
```

```suggestion
GitHubAppCredentials.displayName=GitHub App

```

GitHub uses uppercase in their docs